docs/tutorials/integration/4_databricks/index.md

```diff
@@ -12,222 +12,14 @@ into interactive, production-ready applications for end-users. This article pres
 best practice for integrating both platforms: calling Databricks jobs from a Taipy 
 application. 
```

Comment:
```suggestion
```
This article does not present anything now.

Reply:
Well, for me, the article exists but is not available to the public. But that works for me

Reply:
You already say it below, actually.